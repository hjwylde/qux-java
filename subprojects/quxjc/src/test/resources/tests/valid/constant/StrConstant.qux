str A is 'foo'
str B is 'foo'
str C is 'bar'

str f(str a):
    return a + A

str g(str A):
    return A


void main([str] args):
    print A
    print B
    print C
    print A == B
    print A == C
    print B == C
    print f(C)
    print g(C)

