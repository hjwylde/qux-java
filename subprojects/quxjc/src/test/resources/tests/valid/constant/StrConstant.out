foo
foo
bar
true
false
false
barfoo
bar
